rcl/include/rcl/graph.h

```diff
@@ -401,24 +401,26 @@ RCL_WARN_UNUSED
 rcl_ret_t
 rcl_names_and_types_fini(rcl_names_and_types_t * names_and_types);
 
-/// Return a list of available nodes in the ROS graph.
+/// Return a list of node names and their associated namespaces in the ROS graph.
 /**
  * The `node` parameter must point to a valid node.
  *
- * The `node_names` parameter must be allocated and zero initialized.
- * `node_names` is the output for this function, and contains allocated memory.
+ * The `node_names` and `node_namespaces` parameters must be allocated and zero initialized.
+ * `node_names` and `node_namespaces` are the output for this function, and contain allocated memory.
```

Comment:
```suggestion
 * `node_names` and `node_namespaces` are the output for this function,
 * and contain allocated memory.
```

(the line is > 100 lines, and so will try up our linters)